src/controller.js

```diff
@@ -80,7 +80,7 @@ let Controller = ({
       getGeneEdges({
         modelId: selectedModel,
         geneIds: selectedGenes.map((selected) => selected.id),
-        limit: 100
+        limit: 999999
```

Comment:
This number may not be large enough if you want to load all edges from the backend. The `Number.MAX_SAFE_INTEGER`  constant may be better:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

Reply:
That is the maximum safe javascript integer, but is there a max safe number for django too? Just want to make sure we don't exceed that.

Reply:
Good question. Let's use `9999999` (seven 9's instead of six) for now. Django doc doesn't mention the max int it supports.

Reply:
okay, incorporated comments